doc/ansible/dev/advanced_options.md

```diff
@@ -55,3 +55,59 @@ ENTRYPOINT ["/usr/local/bin/entrypoint", "--inject-owner-ref=false"]
 If you have created resources without owner reference injection, it is
 possible to manually to update resources following [this
 guide.](./retroactively-owned-resources.md)
+
+## Max Workers
+
+Increasing the number of workers allows events to be processed
+concurrently, which can improve reconciliation performance.
+
+Worker maximums can be set in two ways. Operator **authors and admins**
+can set the max workers default by including extra args to the operator
+container in `operator.yaml`. (Otherwise, the default is 1 worker.)
+
+**NOTE:** Admins using OLM should use the environment variable instead
+of the extra args.
+
+``` yaml
+- name: operator
+  image: "quay.io/asmacdo/memcached-operator:v0.0.0"
+  imagePullPolicy: "Always"
+  args:
+    - "--max-workers"
+    - "3"
+```
+
+Operator **admins** can override the value by setting an environment
+variable in the format `WORKER_<kind>_<group>`. This variable must be
+all uppercase, and periods are replaced with underscores.
```

Comment:
> periods are replaced with underscores

It wasn't clear to me until I saw the next snippet of docs that the periods being replaced are the ones that are in the group name. At first, I thought there was a flag name or YAML map key that used something like `worker.<kind>.<group>`.

Maybe add this?
```suggestion
all uppercase, and periods (e.g. in the group name) are replaced with underscores.
```